### DISCUSSION TITLE
Trigger quick terminal like in iterm

### DISCUSSION THREAD

**Discussion by NRVB** [ghostty-org/ghostty#3775] [Feature Requests, Ideas] [answered]
I would like to request a feature that allows users to open the hotkey window by double-tapping the Command (⌘) key, similar to the functionality available in iTerm on macOS.

**Comment by mozarik** [comment]
i think someone in hackernews solve this https://news.ycombinator.com/item?id=42517447 search for `modernerd` and i think you will find the comment

**Reply by NRVB** [reply]
I can only find the option to add a keyboard shortcut, not like iterm that let you to also add the option to double tap cmd/option/shift/ctrl for triggering the hotkey window

**Comment by rhodes-b** [comment]
Ghostty has the ability to have `action` keys 
Please read
https://ghostty.org/docs/config/keybind
https://ghostty.org/docs/config/keybind/sequence
https://ghostty.org/docs/config/reference#keybind

You are probably most particuarly interested in the `global` modifier and `>` sequence described there

https://github.com/ghostty-org/ghostty/discussions/3610
take a look there for getting it to work without the `>` modifier

**Reply by mitchellh** [reply]
I think the issue is they want to trigger it with a modifier double-tap, which we don't support yet.

**Comment by mitchellh** [comment, marked as answer]
Approved #4335
